plop/generators/env.js

```diff
@@ -0,0 +1,18 @@
+export default {
+  description: 'Create environment variables in a .env file',
```

Comment:
I think you should mention that you talk about secret environment variables. There are other ways for adding ordinary environment variables. Or mention it in tocco-docs